### DISCUSSION TITLE
"Only reviewers can change approved strings" but "Enable Reviews" is disabled

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#12069] [General] [answered]
For strings that are "Approved" my translators do see this message.
![Bildschirmfoto_2024-07-16_12-57-23](https://github.com/user-attachments/assets/2fc25df1-38c2-4db8-b151-4198594cc350)

In the beginning "Enable Reviews" was enabled. And after checking the docu I assuming that this was the cause. So I disabled it. But the message still appears.

I don't want to manage permissions for specific users. This increase my maintenance burden.

To my understanding "Approve" is just a quality indicator and shouldn't modify permissions. Every authenticated user should be able to modify every string no matter in what state it is. I do trust my community and also monitor them.

I am assuming that I misunderstood something.

**Comment by nijel** [comment, marked as answer]
Probably this string was approved before, and the approved state is no removed when disabling “Enable reviews”. I think the permission check should ignore the approved state in such case.

If you trust your entire community, you don't need reviews, this is typically used by large communities, with two-level translation workflow: each string is reviewed after being translated. See https://docs.weblate.org/en/latest/workflows.html#dedicated-reviewers

**Reply by nijel** [reply]
Created https://github.com/WeblateOrg/weblate/issues/12075 to track the issue.

**Comment by buhtz** [comment]
Thank you very much.

I see another problem. I bulk-edit all approved strings and set their state to "translated".
No when translating there is only one checkbox "Needs editing" and on "Approved" box anymore. Maybe because I disabled reviews.

I don't want to have the review feature. But I want to have the approve state. It is like a quality indicator that a second translator has reviewed that strings.

**Reply by nijel** [reply]
The review feature is about enabling the approved state and defining who can set that state. Without it, the approved state is not available. You can always define that anybody can approve strings.
